src/elona/profile/profile_manager.cpp

```diff
@@ -3,6 +3,17 @@
 #include "../log.hpp"
 
 
+namespace
+{
+
+void write_default_config(const fs::path& location)
+{
+    std::ofstream out{location.native(), std::ios::binary};
```

Comment:
Since `config.hcl` may be edited by end-user, it should be written in text mode. IIRC, Windows' notepad.exe has not supported LF until recently.